BuildScript.Nuke/GenerateSbom/SolutionSbomGenerator.cs

```diff
@@ -109,6 +114,9 @@ public void Generate (Tool cycloneDxTool)
     {
       Log.Information("No package json file specified, will therefore not create combined sbom.");
 
+      Directory.CreateDirectory(_outputFile.Parent);
```

Comment:
NUKE has `AbsolutePath.CreateDirectory()` so we could write `_outputFile.Parent.CreateDirectory()` instead.